.github/workflows/scan-python.yml

```diff
@@ -0,0 +1,26 @@
+name: Scan Python code with bandit
+'on':
+  pull_request: {}
+  workflow_dispatch: {}
+jobs:
+  bandit:
+    runs-on: ubuntu-latest
+    env:
+      CI: 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: '${{ github.event.pull_request.head.ref }}'
+          repository: '${{ github.event.pull_request.head.repo.full_name }}'
+      - uses: actions/setup-python@v2
```

Comment:
recommend use existing github-actions for bandit env

see https://github.com/awslabs/realtime-fraud-detection-with-gnn-on-dgl/blob/main/.github/workflows/bandit-check.yml as reference

Reply:
changed to use `tj-actions/bandit@v4.1`